app/classes/api2/name_description_api.rb

```diff
@@ -37,9 +37,8 @@ def query_params
         created_at: parse_range(:time, :created_at),
         updated_at: parse_range(:time, :updated_at),
         by_users: parse_array(:user, :user, help: :first_user),
-        names: parse_array(:name, :name),
         is_public: true
-      }
+      }.merge(parse_names_parameters)
```

Comment:
This is a capability addition, rather than a breaking change. It enables NameDescription calls to also parse `include_synonyms` etc., as the API does for observation queries and name queries. 

Necessary, because for Query to be able to convert a Name query to a NameDescription query, `Query::NameDescriptions` must be able to handle the lookup params. That means `NameDescriptionAPI` must be able to send user input as a hash of lookup params to Query. This doesn't change how a user would use the API, it changes how the API packages the user call and sends it to Query.